lib/keyring-manager.ts

```diff
@@ -24,6 +25,10 @@ export class Keyring {
     this.namespace = namespace;
   }
 
+  updateWalletChangeTime() {
+    return storage.setItem(`local:${KEYRING_CHANGE_TIME}`, Date.now());
+  }
```

Comment:
I believe the keyring manager should focus solely on encryption and decryption, and the responsibility for other tasks should be handled elsewhere.